server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -137,6 +138,8 @@ public void registerShuffle(ShuffleRegisterRequest req,
     int shuffleId = req.getShuffleId();
     String remoteStoragePath = req.getRemoteStorage().getPath();
     String user = req.getUser();
+    ShuffleDataDistributionType shuffleDataDistributionType =
```

Comment:
Could we consider the old version client to use the new server?

Reply:
Oh. I missed this point. Let me do it.

Reply:
Done